### PR TITLE
Added conda environment installation

### PR DESCRIPTION
**Pull request by ai-fast-track**
Added `environment.yml` file in order to create a conda environment called `mantis`. I updated the README file by adding the step-by-step instructions including how to handle the `cocapi` installation in  both linux and Windows.

As for now, everything is smoothly installed. For some reason, `fastai2` is not picked up (not installed through this process). Therefore, we have to do `pip install fastai2` from the command line **after activating** the `mantis` conda environment. We have to look into that to figure how to automatically install fastai2.

Once we figure out how to automatically install fastai2, we can add the option to automatically install streamlit.

We have to go through the list of packages in `environment.yml` file and remove the ones that we are not using.

I tested this installation process for both Windows and Linux.

### CODE REVIEW

**Comment by oke-aditya** [comment]
Nice PR @ai-fast-track . Really Nice idea. Cool stuff.
We can skip streamlit for now. Because it takes up a lot of other packages while installing. Streamlit is not necessary for mantisshrimp functionalities

**Comment by ai-fast-track** [comment]
Lightning is automatically installed because it is a Mantisshrimp dependency. The only issue is fastai2. We see how to solve that!

I will check for albumentations: Maybe we can added it as a Mantisshrimp dependency.

**Comment by oke-aditya** [comment]
Maybe we can include fastai v2 in matnisshrimp requirements.txt.
Albumentations is in requirements-extra.txt

**Comment by lgvaz** [comment]
Ow lightning should be removed from our dependencies and then added to `requirements-extra` and the `environment` file =)

File: requirements.txt [hunk 1]
```diff
@@ -4,5 +4,7 @@ tensorboard>=1.14
 pyyaml>=3.13
 fastcore>=0.1.17
 opencv-python>=4.2.0
-pytorch-lightning>=0.7.6
-torchvision==0.6.0
\ No newline at end of file
+torchvision==0.6.0
```
[the remark this answers is not in the document]
Comment (lgvaz) [reply requirements.txt:7]: I'll merge this requirements I will fix it up later

**Comment by lgvaz** [comment]
Actually, when installing from pip, you removed comment saying to install pytorch and torchvision first, I'll ad that

**Comment by lgvaz** [comment]
Github sometimes decides to be insane 🙄🙄

Screenshot 2020-07-09 at 2 39 46 PM

@ai-fast-track can you merge master into this branch locally and push? Maybe it fixes the error

**Comment by ai-fast-track** [comment]
Will do that!

**Comment by lgvaz** [comment]
Closed on #161

**Pull request closed without merging**